applications/smart-distancing/eval/Dockerfile-eval

```diff
@@ -0,0 +1,23 @@
+# docker can be installed by following instructions: 
+# build: docker build -f Dockerfile-amd64usbtpu-eval -t edgetpu-detector-eval .
+# run: docker run -it -v /PATH_TO_CLONED_REPO_ROOT/:/repo detector-eval
+
+FROM python:3.6
+
+WORKDIR /repo/applications/smart-distancing/eval
+
+VOLUME /repo
+
+RUN pip install --upgrade pip setuptools==41.0.0 && pip install six==1.14.0 
+
+RUN pip install --upgrade pip setuptools==41.0.0 && pip install python-opencv==4.2.0.32 && pip install six==1.14.0 
```

Comment:
I tried to build the docker and got this:
ERROR: Could not find a version that satisfies the requirement python-opencv==4.2.0.32 (from versions: none)
ERROR: No matching distribution found for python-opencv==4.2.0.32

Reply:
Did you run it on an x86 device?

Reply:
yes

Reply:
@emma-w-dev I tested the code and didn't get the error.
Let's talk about it on Neuralet slack channel.

Reply:
@emma-w-dev I've tested the code. The error did not occur during the test. 
Let's talk about it on Neuralet slack channel and reproduce the error.

Reply:
Done